.github/workflows/cr.yml

```diff
@@ -0,0 +1,21 @@
+name: Continuous Releases
+
+on:
+  push:
+  pull_request:
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.x
+          cache: yarn
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+      - name: Preview
+        run: npx pkg-pr-new publish --compact
```

Comment:
Is there a way to lock this to a particular version or hash of `pkg-pr-new` so that we have to manually bump it to new versions? Running arbitrary new versions of this in a build script is a major supply chain attack risk.